objects.py

```diff
@@ -14,31 +13,45 @@ def __init__():
 
         # This should set the object's price and calories, accessible by
         # .price and .calories respectively.
+        """ For creating desserts """
+        self.price = price
+        self.calories = calories
         pass
```

Comment:
`pass` is really just a placeholder when you want to have something inside an indent but haven't written what it is yet. once you have other code in there, you can delete the `pass`.